### PR TITLE
Readable json + updated models examples 

### PR DESCRIPTION
**Pull request by glimow**
must be merged after https://github.com/src-d/models/pull/18

### CODE REVIEW

File: bow/1e0deee4-7dc1-400f-acb6-74c0f4aec471.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ Example:
 
 ```
 from sourced.ml.models import BOW
-bow = BOW().load("1e0deee4-7dc1-400f-acb6-74c0f4aec471")
+import modelforge.models
```
Comment (irinakhismatullina) [inline bow/1e0deee4-7dc1-400f-acb6-74c0f4aec471.md]:
```suggestion
import modelforge.backends
```

Comment (irinakhismatullina) [reply bow/1e0deee4-7dc1-400f-acb6-74c0f4aec471.md]: And in all the other places

**Review by irinakhismatullina — CHANGES_REQUESTED** [review]

**Comment by glimow** [comment]
@irinakhismatullina @vmarkovtsev PTAL

File: README.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ Example:
 
 ```python
 from sourced.ml.models import BOW
-bow = BOW().load(bow)
+import modelforge.backends
```
Comment (vmarkovtsev) [inline README.md]: No longer needed after https://github.com/src-d/modelforge/pull/92

File: index.json [hunk 3]
```diff
@@ -1 +1,410 @@
...
\ No newline at end of file
+{
```
Comment (vmarkovtsev) [inline index.json:1]: Can you please split this PR into two:

1. Readable JSON, no other changes.
2. Extra swagger you add to JSON and generate to Markdown

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**